src/test/resources/org/jabref/logic/importer/fetcher/collection_of_computer_science_bibliographies_single_result.bib

```diff
@@ -0,0 +1,14 @@
+@inproceedings{conf/zeus/SimonDDK19,
+  author = {Martin K. Simon and Linus W. Dietz and Tobias Diez and Oliver Kopp},
+  bibdate = {2019-05-28},
+  bibsource = {DBLP, http://dblp.uni-trier.de/db/conf/zeus/zeus2019.html#SimonDDK19},
+  booktitle = {Proceedings of the 11th Central European Workshop on Services and their Composition, Bayreuth, Germany, February 14-15, 2019},
+  editor = {Stefan Kolb 0001 and Christian Sturm 0002},
```

Comment:
Oh, wow, the `0001` and `0002` are generated by CCSB?

Would it be possible to remove `[ ]\d+` at editors?

Excpected result:

    Stefan Kolb and Christian Sturm